packages/flutter/lib/src/gestures/converter.dart

```diff
@@ -104,90 +93,44 @@ class PointerEventConverter {
           );
           break;
```

Comment:
We should also document the legal sequence of events somewhere. Ideally, we add the diagram describing the legal sequence to https://github.com/flutter/assets-for-api-docs and link it in an appropriate space.